index.html

```diff
@@ -53,5 +53,167 @@ <h2 class="section-header-subtitle">I’m a Software Developer</h2>
+           <h1 class="art-data">Profesional Art Printing Data</h1> 
+            <p class="daily-s-con">A daily selection of privately personalized reads;
+              no accounts or sign-ups required. has been the industry's standard
+            </p>
+          </div>
+          <ul class="pro-t">
+            <li class="pro-">html</li>
+            <li class="pro-">bootstrap</li>
+            <li class="pro-">Ruby</li>
+          </ul>
+             <button class="see-project" type="button">See Project</button>
+        </div>
+        <div class="pro-art-major">
+          <div class="art">
+           <h1 class="art-data">Profesional Art Printing Data</h1> 
+            <p class="daily-s-con">A daily selection of privately personalized reads;
+              no accounts or sign-ups required. has been the industry's standard
+            </p>
+          </div>
+          <ul class="pro-t">
+            <li class="pro-">html</li>
+            <li class="pro-">bootstrap</li>
+            <li class="pro-">Ruby</li>
+          </ul>
+             <button class="see-project" type="button">See Project</button>
+        </div>
+        <div class="pro-art-major">
+          <div class="art">
+           <h1 class="art-data">Profesional Art Printing Data</h1> 
+            <p class="daily-s-con">A daily selection of privately personalized reads;
+              no accounts or sign-ups required. has been the industry's standard
+            </p>
+          </div>
+          <ul class="pro-t">
+            <li class="pro-">html</li>
+            <li class="pro-">bootstrap</li>
+            <li class="pro-">Ruby</li>
+          </ul>
+             <button class="see-project" type="button">See Project</button>
+        </div>
+        <div class="pro-art-major">
+          <div class="art">
+           <h1 class="art-data">Profesional Art Printing Data</h1> 
+            <p class="daily-s-con">A daily selection of privately personalized reads;
+              no accounts or sign-ups required. has been the industry's standard
+            </p>
+          </div>
+          <ul class="pro-t">
+            <li class="pro-">html</li>
+            <li class="pro-">bootstrap</li>
+            <li class="pro-">Ruby</li>
+          </ul>
+             <button class="see-project" type="button">See Project</button>
+        </div>
+
+
+
+      </section>
+        <section class="about-general">
+          <div class="about-me">
+            <div class="about-me-gen">
+              <h2 class="about-header">About me</h2>
+              <p class="hello-software">Hello I’m a software developer! I can help you build a product , feature or website Look through
+                some of my work and experience! 
+                If you like what you see and have a project you need coded, don’t hestiate to contact me.
+              </p>
+              <button class="button" type="button">Get My Resume</button>
+            </div>
+          </div>
```

Comment:
- [ ] Good job, you are getting there, please fix the background problem with this section, here's the difference:

Template:
![Captura de pantalla 2022-03-18 a la(s) 9 22 13 p  m](https://user-images.githubusercontent.com/83717285/159103165-bea1ff13-8374-49ff-8a38-47e12e7dc93e.png)

Your design:
![Captura de pantalla 2022-03-18 a la(s) 9 21 57 p  m](https://user-images.githubusercontent.com/83717285/159103175-6483eb8b-3dff-435b-b728-ad2105828109.png)

Reply:
fixed